tests/src/test/java/org/eclipse/hono/tests/mqtt/MqttConnectionIT.java

```diff
@@ -143,4 +147,41 @@ public void testConnectFailsForDeletedDevices(final TestContext ctx) {
                         ((MqttConnectionException) t).code());
             }));
     }
+
+
+    /**
+     * Verifies that the adapter rejects connection attempts from devices
+     * that belong to a tenant for which the adapter has been disabled.
+     *
+     * @param ctx The test context
+     */
+    @Test
+    public void testConnectFailsForDisabledTenant(final TestContext ctx) {
```

Comment:
this seems to be a duplicate of *testConnectFailsForDisabledAdapter*, isn't it?
However, it might make sense to have a test that actually verifies that a device belonging to a disabled *tenant* cannot connect, i.e. 
```
final TenantObject tenant = TenantObject.from(tenantId, false);
```

Reply:
Its a good catch. That was an existing test case and I just moved to MqttConnectionIT class. 
I will enable the adapter and disable the tenant.